src/commands/updates/browser.ts

```diff
@@ -24,15 +24,22 @@ import {
  * https://searchfox.org/mozilla-central/source/taskcluster/gecko_taskgraph/util/partials.py
  */
 const ausPlatformsMap = {
-  linux64: ['Linux_x86_64-gcc3'],
-  macosIntel: [
+  linux: [
+    'Linux_x86_64-gcc3',
+    'Linux_aarch64-gcc3',
+  ],
+  macos: [
     'Darwin_x86_64-gcc3-u-i386-x86_64',
     'Darwin_x86-gcc3-u-i386-x86_64',
     'Darwin_x86-gcc3',
     'Darwin_x86_64-gcc3',
+    'Darwin_aarch64-gcc3',
+  ],
+  windows: [
```

Comment:
It doesn't since the update url is based on the target
Ex. Linux aarch64 would go to updates/browser/**Linux_aarch64-gcc3**/**twilight-aarch64**/update.xml